istio-control/istio-discovery/templates/configmap-envoy.yaml

```diff
@@ -47,7 +47,57 @@ data:
               max_pending_requests: 100000
               max_requests: 100000
               max_retries: 3
-
+        hosts:
+          - socket_address:
+              address: istio-galley.{{ .Values.global.configNamespace }}
+              port_value: 15019
+      {{- if .Values.global.controlPlaneSecurityEnabled }}
+      {{- if .Values.global.sds.enabled }}
+        tls_context:
+          common_tls_context:
+            tls_certificate_sds_secret_configs:
+            - name: default
+              sds_config:
+                api_config_source:
+                  api_type: GRPC
+                  grpc_services:
+                  - google_grpc:
+                      target_uri: {{ .Values.global.sds.udsPath }}
+                      channel_credentials:
+                        local_credentials: {}
+                      call_credentials:
+                      - from_plugin:
+                          name: envoy.grpc_credentials.file_based_metadata
+                          config:
+                            header_key: istio_sds_credentials_header-bin
+                            secret_data:
+                              filename: {{ .Values.global.sds.token.path }}
+                      credentials_factory_name: envoy.grpc_credentials.file_based_metadata
+                      stat_prefix: sdsstat
+            combined_validation_context:
+              default_validation_context:
+                verify_subject_alt_name:
+                - spiffe://{{ .Values.global.trustDomain }}/ns/{{ .Values.global.configNamespace }}/sa/istio-galley-service-account
```

Comment:
duplicated at 112...

Reply:
This is not a duplicate. Each one for a if-else branch